solutions/chapter8/services/theme.ts

```diff
@@ -0,0 +1,196 @@
+import {
+  DarkTheme,
+  DefaultTheme,
+  useTheme as useNavTheme,
+} from '@react-navigation/native'
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react'
+import type { StyleProp } from 'react-native'
+import { Appearance, useColorScheme } from 'react-native'
+
+import { changeHexAlpha, colors as lightColors } from '../../../shared/theme'
+
+const darkColors = {
+  background: {
+    primary: '#000000',
+    secondary: '#1a1a1a',
+    accent: '#099faa',
+    accentMuted: '#f8baf7',
+    brand: '#d4f20d',
+    reflection: '#ffffff',
+    transparent: '#000000',
+  },
+  text: {
+    base: '#ffffff',
+    accent: '#f8baf7',
+    brand: '#d4f20d',
+    baseMuted: '#4d4d4d',
+    overlay: '#000000',
```

Comment:
Because this property is used over images, which won't change with the appearance mode, we should consider keep this as `#ffffff`

Reply:
![CleanShot 2025-01-02 at 12 58 11@2x](https://github.com/user-attachments/assets/48f08366-9ed4-41f4-afb9-d4ef0b891c30)

Reply:
fixing